ee/src/main/java/org/jboss/as/ee/component/InterceptorEnvironment.java

```diff
@@ -26,7 +26,7 @@
 import java.util.List;
 
 /**
- * Class that represents the environment entries of an interceptor, defined in the interceptors section
+ * Class that represents the environment entries of an interceptor, defined in the Jakarta Interceptors section
```

Comment:
This comment is making a reference to a section on the ejb-jar.xml, I've my doubts about whether should be changed, @asoldano Do you have any opinion about it?

Reply:
For clarity, I just suggest replacing it with " ... defined in the <interceptors> section ..."

Reply:
if the section is named "interceptors", we would leave this as is, I think..